src/engraving/dom/spanner.h

```diff
@@ -212,7 +212,7 @@ class Spanner : public EngravingItem
     PropertyValue propertyDefault(Pid propertyId) const override;
     virtual void undoChangeProperty(Pid id, const PropertyValue&, PropertyFlags ps) override;
 
-    void computeStartElement();
+    virtual void computeStartElement();
     void computeEndElement();
```

Comment:
Shouldn't it be virtual as well?

Reply:
At the moment, there is no item that overrides `computeEndElement` (I only needed to override `computeStartElement` , and only for the Trill class because of its particular need of managing the Ornament object). Shall I mark it as virtual anyway for consistency?

Reply:
Not too important right now... You can mark it as virtual in your next PR